### ISSUE TITLE
[HELP!!!]  Changing the resolution during model training results in NAN values.

### ISSUE THREAD

**Issue by MaverickRen** [NVIDIA/Cosmos#153]
A severe bug has occurred: changing the resolution during model training results in NAN values. As of the code up to March 24th, when training with images of 320x512 resolution (33 frames totaling 3200 tokens), whether loading pre-trained weights or initializing randomly, the logits become NAN during inference.

**Comment by mingyuliutw** [comment]
The code in this repo will be deprecated. We are actively developing under https://github.com/nvidia-cosmos
For NeMo users, please visit https://github.com/NVIDIA/NeMo
